ui/src/components/symbols.tsx

```diff
@@ -8,7 +8,7 @@ export class Symbols extends Component<any, any> {
 
   render() {
     return (
-      <svg aria-hidden="true" style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+      <svg aria-hidden="true" style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
```

Comment:
That one attribute is invalid JSX. I think it just got stripped out when compiled, but ESLint chokes on it.

Reply:
Oh sweet, thx.